dip-template/runtimes/dip-provider/src/lib.rs

```diff
@@ -261,6 +235,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type SelfParaId = ParachainInfo;
 	type XcmpMessageHandler = ();
+	type ConsensusHook = cumulus_pallet_parachain_system::ExpectParentIncluded;
```

Comment:
fixed velocity

Reply:
fixed in: [ca13129](https://github.com/KILTprotocol/kilt-node/pull/627/commits/ca13129273c0abcfb2dbeea57c5efeaab730693a)